wwwroot/inc/snmp.php

```diff
@@ -2069,6 +2069,34 @@
 	'try_next_proc' => FALSE,
 );
 
+$iftable_processors['brocade-vdx-QSFP+'] = array
+(
+	'pattern' => '@^FortyGigabitEthernet 0/(\d+)$@',
+	'replacement' => 'Fo /0/\\1',
```

Comment:
I mean, could you confirm which of the following variants the CLI accepts as a valid interface name?
* `fo /0/0/1`
* `fo 0/0/1`
* `fo0/0/1`